src/components/display/DisplayImage.js

```diff
@@ -0,0 +1,40 @@
+import { imageURL } from "../api/axios";
+import { useState, useEffect } from "react";
+import "./DisplayImage.css";
+
+const DisplayImage = () => {
+  const [images, setImages] = useState([]);
+
+  const showImage = async () => {
+    try {
+    const res = await imageURL
+      setImages(res.data.results);
+      console.log(res.data.results);
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+  useEffect(() => {
+    showImage();
+  }, []);
```

Comment:
I feel `showImage` definition should be inside the `useEffect`

Reply:
ok